### DISCUSSION TITLE
admin/diagnostics page returns 404 in latest version (1.26.0)

### DISCUSSION THREAD

**Discussion by yggdrasil-tynor** [dani-garcia/vaultwarden#2881] [Q&A]
### Subject of the issue
admin/diagnostics returns a 404 in latest version (1.26.0). 

### Deployment environment
Docker in Azure App Service (cannot post diagnostics as it is not working anymore)

* vaultwarden version:
1.26.0

* Install method:
docker run @ Azure

* Clients used: 
web client

* MySQL/MariaDB or PostgreSQL version: 
Embedded database (sqlite)

### Steps to reproduce
Running latest tag of vaultwarden/server:alpine

### Expected behaviour
Show the diagnostics page

### Actual behaviour
404 site 

### Troubleshooting data

> 2022-11-01T11:06:01.448767120Z [2022-11-01 11:06:01.447][request][INFO] GET /admin/diagnostics
> 2022-11-01T11:06:01.449452230Z [2022-11-01 11:06:01.447][_][WARN] Request guard `AdminToken` is forwarding.
> 2022-11-01T11:06:01.449464530Z [2022-11-01 11:06:01.447][_][WARN] Response was `None`.
> 2022-11-01T11:06:01.449469030Z [2022-11-01 11:06:01.447][_][WARN] Responding with registered (not_found) 404 catcher.
> 2022-11-01T11:06:01.449473230Z [2022-11-01 11:06:01.448][response][INFO] (web_files) GET /<p..> [10] => 404 Not Found

**Comment by BlackDex** [comment]
You need to go to `/admin` first, login, and after that you can go to the diagnostics page.
